### ISSUE TITLE
i love you

### ISSUE THREAD

**Issue by EisregenHaha** [xor2k/gpu_undervolt#6]
man i genuinely came all over my screen finally i can dily drive fedora. 
although the line 163:    DISPLAY=$(cd /tmp/.X11-unix && for x in X*; do echo ":${x#X}"; done) \
causes an error for me saying the display isnt found, after deleting that line i could use the script and it works. it it bad that i had to delete the line?

**Comment by xor2k** [comment]
That's only relevant if you want to run the script at bootup e.g. by `/etc/rc.local`. If you run it as user, that line should not be needed.

You're welcome!

**Comment by EisregenHaha** [comment]
> That's only relevant if you want to run the script at bootup e.g. by `/etc/rc.local`. If you run it as user, that line should not be needed.
> 
> You're welcome!

yeah i just created a service running the script on bootup after this so its fine. thanks
